plan/builders.go

```diff
@@ -286,11 +287,13 @@ func (b *builder) AggregateColumnsRemap(input Rel, remap []int32, measures []Agg
 		}
 	}
 
+	groupingExpressions, groupingReferences := groupingExprs(exprs)
```

Comment:
We could ditch the Remap versions since they've been deprecated for a while (hehe, for at least two major versions) to avoid needing passing these changes along.